src/main/java/frc/robot/subsystems/hood/Hood.java

```diff
@@ -49,6 +54,10 @@ public MutableMeasure<Angle> getAngle() {
         return inputs.angle;
     }
 
+    public MutableMeasure<Voltage> getVoltage() {
+        return inputs.voltage;
+    }
+
```

Comment:
Why is that needed?

Reply:
It was needed for the test we removed
0d14d35